gimmemotifs/orthologs.py

```diff
@@ -167,45 +173,72 @@ def _orthofinder(peptide_folder, threads):
 
 
 def _download_genomes_with_annot(genomes, genomes_dir):
-    # make sure each genome has a gene annotation
-    no_annotations = [genome for genome in genomes if not _has_annotation(genome)]
-    assert (
-        len(no_annotations) == 0
-    ), f"genome(s): {','.join(no_annotations)} seem not to have an annotation for it."
-
     # download the genomes
     # add check to see if not already in genomes dir?
     default_genomes_dir = get_genomes_dir(check_exist=False)
     logger.debug(f"using default genome dir: {default_genomes_dir}")
+    existing_genomes = []
     for genome in genomes:
         # check if already in default genomes dir, if so, skip downloading and directly copy
         if all(
-            os.path.exists(f"{default_genomes_dir}/{genome}/{genome}.{extension}")
+            os.path.exists(f"{default_genomes_dir}/{genome}/{genome}.{extension}") or 
+            os.path.exists(f"{default_genomes_dir}/{genome}/{genome}.{extension}.gz")
             for extension in ["fa", "annotation.gtf"]
         ):
             logger.info(f"{genome} was already downloaded, using that version.")
             # if except, probably a continuation from previous run
             try:
                 os.mkdir(f"{genomes_dir}/{genome}")
-                os.symlink(
-                    f"{default_genomes_dir}/{genome}/{genome}.fa",
-                    f"{genomes_dir}/{genome}/{genome}.fa",
-                )
-                os.symlink(
-                    f"{default_genomes_dir}/{genome}/{genome}.annotation.gtf",
-                    f"{genomes_dir}/{genome}/{genome}.annotation.gtf",
-                )
+                pep = f"{default_genomes_dir}/{genome}/{genome}.pep.fa"
+                if os.path.exists(pep):
+                    logger.info(f"found peptide fa for {genome}")
+                    os.symlink(
+                        pep,
+                        f"{genomes_dir}/{genome}/{genome}.pep.fa",
+                    )
```

Comment:
p.s. no relevance to the correctness of the code, but the function is called `_download_genomes_with_annot` so it might be a bit surprising to the flow when this function does not download a genome based on that a peptide already exists.

Reply:
Ah, so ideally that check should then be moved outside that function.